File: RxTools-library/src/main/java/com/vondear/rxtools/model/ActionItem.java
package com.vondear.rxtools.model;

/**
 * @author Vondear
 * 功能描述：弹窗内部子类项（绘制标题和图标）
 */
public class ActionItem {
    /**
     * 定义文本对象
     */
    public CharSequence mTitle;

    public int mResourcesId;

    public ActionItem(CharSequence title, int mResourcesId) {
        this.mResourcesId = mResourcesId;
        this.mTitle = title;
    }

    public ActionItem(CharSequence title) {
        this.mTitle = title;
    }

}

File: RxKit/src/main/java/com/tamsiree/rxkit/model/ActionItem.kt
package com.tamsiree.rxkit.model

/**
 * @author Tamsiree
 * 功能描述：弹窗内部子类项（绘制标题和图标）
 */
class ActionItem {
    /**
     * 定义文本对象
     */
    @JvmField
    var mTitle: CharSequence

    @JvmField
    var mResourcesId = 0

    constructor(title: CharSequence, mResourcesId: Int) {
        this.mResourcesId = mResourcesId
        mTitle = title
    }

    constructor(title: CharSequence) {
        mTitle = title
    }
}